routing/pathfind.go

```diff
@@ -819,6 +819,7 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 				lnwire.TLVOnionPayloadOptional,
 			)
 
+			// TODO: this is now incorrect?
```

Comment:
Interesting Point, I think we need to add the encrypted data here to make sure it does not exceed the maximum sphinx size?

Reply:
I'm tempted to fix this in a follow up (because touching findpath is scary), and we're already _technically_ incorrect here because we don't include things like metadata / MPP fields (and in practice we're nowhere near hitting this limit). WDYT?

Reply:
Sounds good, let's fix it in the follow-up PR.